micropython/main.py

```diff
@@ -85,7 +86,11 @@
 
 # get X.509 certificate from SIM
 csr = ubirch.get_certificate(device_name)
-print("X.509 certificate: " + binascii.hexlify(csr).decode())
+print("X.509 certificate: " + binascii.b2a_base64(csr).decode().rstrip('\n'))
+
+import sys
```

Comment:
Why is the import here?

Reply:
i just used sys.exit() to exit the program at this point during developing. i took it out now.